unime/.gitignore

```diff
@@ -1,35 +1,16 @@
-# Logs
-logs
-*.log
-npm-debug.log*
-yarn-debug.log*
-yarn-error.log*
-pnpm-debug.log*
-lerna-debug.log*
-
-/node_modules
+# .gitignore specific to /unime
```

Comment:
See comment in `identity-wallet/.gitignore`

Reply:
Comment removed.